<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YOLO NPU 目标检测</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_left">
      <item>
       <widget class="QGroupBox" name="groupBox_Video">
        <property name="title">
         <string>视频窗口</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_video">
         <item>
          <widget class="QLabel" name="videoLabel">
           <property name="styleSheet">
            <string notr="true">background-color: #f0f0f0;</string>
           </property>
           <property name="text">
            <string>请选择视频</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_right">
      <item>
       <widget class="QGroupBox" name="groupBox_Detect">
        <property name="title">
         <string>检测窗口</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_detect">
         <item>
          <widget class="QLabel" name="detectLabel">
           <property name="styleSheet">
            <string notr="true">background-color: #f0f0f0;</string>
           </property>
           <property name="text">
            <string>检测结果将显示在这里</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>174</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="chooseVideoBtn">
            <property name="text">
             <string>选择视频</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playBtn">
            <property name="text">
             <string>播放/暂停</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="detectBtn">
            <property name="text">
             <string>检测当前帧</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveBtn">
            <property name="text">
             <string>保存结果</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
